pages/en/glossary.mdx

```diff
@@ -0,0 +1,81 @@
+
+- **Indexer's self stake**: The amount of GRT that Indexers stake to process subgraphs on the network. The minimum is 100,000 GRT, and there is no upper limit.
+
+- **Delegators**: Individuals who own GRT and stake (delegate) their GRT to Indexers. This allows Indexers to increase their stake in subgraphs on the network. In return, Delegators receive a portion of the Indexing Rewards that Indexers receive for processing subgraphs.
+
+- **Delegation tax**: A 0.5% fee paid by Delegators when they delegate GRT to Indexers. The GRT used to pay the fee is burned.
+
+- **Curators**: Curators identify high-quality subgraphs, and “curate” them (i.e., signaling GRT on them) to earn curation shares, which guarantee a percentage of all future query fees generated by the subgraph. Curating a subgraphs increases the chances of a subgraph being indexed by an Indexer.
+
+- **Curation tax**: A 1% fee paid by Curators when they signal GRT on subgraphs. The GRT used to pay the fee is burned.
+
+- **Subgraph consumers**: Any application or user that queries a subgraph.
+
+- **Subgraph developer**: A developer who builds and deploys a subgraph to The Graph's decentralized network.
+
+- **Subgraph manifest**: A JSON file that describes the subgraph's GraphQL schema, data sources, and other metadata.
+
+- **Rebate pool**: An economic security measure that holds query fees paid by subgraph consumers. Indexers must wait for an epoch to close before they can claim their query fee rebates.
+
+- **Epoch**: A period of time that is used to measure the performance of Indexers. At the end of an epoch, Indexers are rewarded for their performance.
+
+- **Allocation**: An Indexer can allocate self and delegated stake towards subgraphs that have been published on The Graph's decentralized network. This is called opening an allocation, and indicates to the network that the Indexer is actively indexing and serving queries to the subgraph. Open allocations accrue indexing rewards proportional to the signal on the subgraph, and the amount of GRT allocated. An Indexer may claim the accrued indexing rewards on a given subgraph by submitting a recent, and valid, Proof of Indexing (POI). This is known as closing an allocation. The maximum allocation period is 28 epochs. If an indexer leaves an allocation open beyond 28 epochs, it is known as a stale allocation. Allocations exist in one of four phases:
+
+  1. **Active**: An allocation is considered active when it is created on-chain. The Indexer agent manages creating allocations based on the Indexer rules.
+
+  2. **Closed**: An Indexer can close an allocation once an epoch has passed, or their Indexer agent will automatically close the allocation after ~26 days. When an allocation is closed, their indexing rewards are distributed to the Indexer and its Delegators.
+
+  3. **Finalized**: Once an allocation has been closed, there is a dispute period, allowing an opportunity for a fisherman to open a dispute.
```

Comment:
This is not 100% clear to me: can fishermen open a dispute in the "finalized" phase, or _before_ that phase?

Reply:
Cleared that up! Thank you for pointing that out!